src/app/templates/DropdownSelect/DropdownSelect.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { ChangeEventHandler, ReactNode, FC, Dispatch, SetStateAction } from 'react';
+
+import { isDefined } from '@rnw-community/shared';
+import classNames from 'clsx';
+
+import AssetField from 'app/atoms/AssetField';
+import DropdownWrapper from 'app/atoms/DropdownWrapper';
+import Spinner from 'app/atoms/Spinner/Spinner';
+import { ReactComponent as ChevronDownIcon } from 'app/icons/chevron-down.svg';
+import { ReactComponent as SearchIcon } from 'app/icons/search.svg';
+import { AnalyticsEventCategory, setTestID, useAnalytics } from 'lib/analytics';
+import { t } from 'lib/i18n';
+import Popper from 'lib/ui/Popper';
+import { sameWidthModifiers } from 'lib/ui/same-width-modifiers';
+
+interface Props<T> {
+  DropdownFaceContent: ReactNode;
+  Input?: ReactNode;
+  optionsListClassName?: string;
+  dropdownButtonClassName?: string;
+  searchProps: SelectSearchProps;
+  optionsProps: SelectOptionsPropsBase<T>;
+  testIds?: {
+    dropdownTestId?: string;
+    searchInputTestId?: string;
+  };
+}
+
+export const DropdownSelect = <T extends unknown>({
+  Input,
+  searchProps,
+  optionsProps,
+  testIds,
+  DropdownFaceContent,
+  optionsListClassName,
+  dropdownButtonClassName
+}: Props<T>) => {
+  const isInputDefined = isDefined(Input);
+  const { trackEvent } = useAnalytics();
+
+  const trackDropdownClick = () => {
+    if (testIds?.dropdownTestId) {
+      trackEvent(testIds?.dropdownTestId, AnalyticsEventCategory.DropdownOpened);
+    }
+  };
+
+  return (
+    <Popper
+      placement="bottom"
+      strategy="fixed"
+      modifiers={sameWidthModifiers}
+      fallbackPlacementsEnabled={false}
+      popup={({ opened, setOpened }) => (
+        <SelectOptions
+          optionsListClassName={optionsListClassName}
+          opened={opened}
+          setOpened={setOpened}
+          {...optionsProps}
+        />
+      )}
+    >
+      {({ ref, opened, toggleOpened }) => (
+        <div ref={ref as unknown as React.RefObject<HTMLDivElement>}>
+          {opened ? (
+            <SelectSearch className={dropdownButtonClassName} {...searchProps} />
+          ) : (
+            <div
+              className="box-border w-full flex items-center justify-between border rounded-md border-gray-300 overflow-hidden max-h-18"
+              style={{ maxHeight: '4.5rem' }}
+            >
+              <button
+                className={classNames(
+                  'flex gap-2 items-center',
+                  isInputDefined && 'border-r border-gray-300',
+                  !isInputDefined && 'w-full justify-between',
+                  dropdownButtonClassName
+                )}
+                onClick={() => {
+                  toggleOpened();
+                  trackDropdownClick();
+                }}
+                style={{ maxHeight: '4.5rem' }}
```

Comment:
can we move `maxHeight: '4.5rem'` to tailwind config and use in className?

Reply:
`max-h-18` doesn’t work. I don’t know why

Reply:
you need to add this value to `tailwind.config.js`, now it existed only for minHeight, not maxHeight